TombEngine/Game/effects/spark.cpp

```diff
@@ -140,4 +142,32 @@ namespace TEN::Effects::Spark
 			s.active = true;
 		}
 	}
+
+	void TriggerAttackSpark(SparkOffsets* pose, Vector3 color)
```

Comment:
Also colour should be a const reference: `const Vector3& color`.

Reply:
changed